schema.dbml

```diff
@@ -990,3 +981,33 @@ Table susc_results as Susc {
     (rx_name, control_iso_name, iso_name, ref_name, potency_type, assay_name) [pk]
   }
 }
+
+Table invivo_selection_results {
```

Comment:
The new `invivo_selection_results` becomes the derived table using the data from `rx_invivo`, `subjects`, `subject_history` and `variant_mutations`. I tried to preserve as many old fields as possible but following our established naming convention.

Reply:
what is "variant_mutations"? How the *position* and *amino_acid* field in *invivo_selection_results* generated by other tables? What if the paper doesn't provide isolates from GISAID by only single point mutations, how to enter the data?